src/Avalonia.Controls/ToolTipService.cs

```diff
@@ -88,8 +88,9 @@ private void ControlDetaching(object? sender, VisualTreeAttachmentEventArgs e)
         private void ControlPointerEntered(object? sender, PointerEventArgs e)
         {
             StopTimer();
-
             var control = (Control)sender!;
+            e.Pointer.Capture(control);
```

Comment:
I'm not sure that capturing the mouse when entering a control with a tooltip is a good idea. What if the pointer has already been captured for another purpose for example?

Reply:
I agree, this will break things.

A few things that aren't working correctly anymore with a simple test:
 - `TextBox` cursor keeps switching from caret to arrow when it has a tooltip.
 - Can't have a tooltip over a popup anymore (I've tested with a `ComboBoxItem`), probably capture fighting.

More advanced scenarios involving captures might also be broken, but the simple two cases above are enough to show that this approach probably adds more problems than it solves.

Reply:
I am glad you were able to solve it in a better way!